### PR TITLE
renamed wishlist to want_to_play, currently playing to my_games, frie…

### PR DESCRIPTION
**Pull request by MonikaTheiss**
renamed wishlist to want_to_play, currently playing to my_games, friends to connections, friend_requests to connection_requests, updated default profile to use this naming. Updated the database as well.

### CODE REVIEW

File: lib/view/pages/connections/connections_page.dart [hunk 1]
```diff
@@ -177,7 +177,8 @@ class _FriendSearchState extends State<FriendSearch> {
         title: const Text('Friends'),
```
Comment (pieryvdl) [inline lib/view/pages/connections/connections_page.dart:177]: We should maybe also make this text 'connections'.

Comment (LeratoKgomo) [reply lib/view/pages/connections/connections_page.dart:177]: this a good idea

**Review by pieryvdl — APPROVED** [review]
Renaming looks great Monika! Thank you for spending time on this and fixing it. I left a comment of where you possibly missed a renaming.

**Review by LeratoKgomo — APPROVED** [review]

File: lib/view/pages/connections/connections_page.dart (hunk 1 above)
Comment (MonikaTheiss) [reply lib/view/pages/connections/connections_page.dart:177]: I will quickly change it. I got pretty confused at a stage with all the renaming 😅

**Review by pieryvdl — APPROVED** [review]

**Pull request merged**